config/protocol.unit_testnet.yml

```diff
@@ -19,6 +19,9 @@ ProtocolConfiguration:
   P2PSigExtensions: true
   Hardforks:
     Aspidochelone: 25
+    Basilisk: 41
+    Cockatrice: 55
+    Domovoi: 60
```

Comment:
Heights are too large, it may lead to useless block addition in tests like `neoValidatorInvoker.AddNewBlock(t)`. In general, I'd like all forks to be enabled before Basic chain height, something like `A: 3, B: 6, C:9...`, so that in future we're able to add some tests for forks-related code using our basic chain. But at the same time I remember that there was a pitfall with `Aspidochelone: 25`, there was a reason why did we use `25`.

But in general it should be possible, some unit-tests may be fixed after this change.